app/facades/pro.rb

```diff
@@ -0,0 +1,70 @@
+class Pro < Dashboard
+  def initialize(user)
+    super
+  end
+
+  def user_article_ids
+    user.articles.pluck(:id)
```

Comment:
I suggest memoizing like `@user_articlle_ids ||= user.articles.pluck(:id)`. The method is called multiple times, so with memoization, the values won't need to be arranged multiple times.

Reply:
TIL!